### PR TITLE
Merge in faucet-admin

### PR DESCRIPTION
**Pull request by jnordberg**
This brings in [faucet-admin](https://github.com/steemit/faucet-admin)

To run faucet-admin start server with the env var `FAUCET_ADMIN` set

Closes https://github.com/steemit/faucet-admin/issues/46

### CODE REVIEW

**Comment by jnordberg** [comment]
@jredbeard the only deployment change needed should be to switch the image pulled in and adding `FAUCET_ADMIN=1` to configuration, in faucet-admin. faucet deployment will remain the same

File: admin/package.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
```
[the remark this answers is not in the document]
Comment (bnchdrff) [reply admin/package.json]: also we should do this to `.env.example`:

```diff
diff --git a/.env.example b/.env.example
index 65bb331..8451c12 100644
--- a/.env.example
+++ b/.env.example
@@ -14,4 +14,11 @@ CREATE_USER_URL=https://steemit.com/api/create_user
 LOG_LEVEL=debug
 PORT=3000
 STEEMJS_URL=https://api.steemit.com
-DEFAULT_REDIRECT_URI=https://steemit.com
\ No newline at end of file
+DEFAULT_REDIRECT_URI=https://steemit.com
+FAUCET_ADMIN=
+COOKIE_SECRET=123
+DATABASE_URL=mysql://foo:bar@host/dbname
+FAUCET_URL=https://signup.steemit.com
+GOOGLE_CALLBACK=https://faucetadmin/auth/google/callback
+GOOGLE_CLIENT_ID=topsecret.apps.googleusercontent.com
+GOOGLE_CLIENT_SECRET=evenMoreTopSecret

```

[the remark this answers is not in the document]
Comment (bnchdrff) [reply admin/package.json]: and add instructions to README

[the remark this answers is not in the document]
Comment (jnordberg) [reply admin/package.json]: good point, I'll merge the two package.json deps

**Comment by jnordberg** [comment]
Back to you @bnchdrff

**Pull request merged**